main.cpp

```diff
@@ -107,11 +108,52 @@ struct CarWash
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
+struct Person
+{
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTraveled;
+
+struct Foot
+    {
+    float length;
+    int toes = 5;
+    bool flatFooted;
+    bool isLeft = true;
 
+    void stepForward();
+    int stepSize();
+    };
 
+    void run(int howFast, bool startWithLeftFoot);
+};
 
+void Person::Foot::stepForward()
+{
+    if(isLeft==true)
+    {
+        std::cout << "Left!" << std::endl;
+    }
+    else
+    {
+        std::cout << "Right!" << std::endl;
+    }
+}
 
-
+int Person::Foot::stepSize()
+{
+    if(flatFooted == true)
+    {
+        return 5;
+    }
+    else
+    {
+        return 8;
+    }
+}
```

Comment:
If you find yourself writing code like this:
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    else 
    {
        return anotherValue;
    }
}
```
You do not need the `else` block.  Here's why:
`return` statements are the last thing that **ever happens** in a function.
When the `if()` expression is evaluated as `true` and the block is entered, the `return aValue;` is the last line of code to be executed before the function exits.
but when the `if()` expression evaluates as `false` and the block is **NOT** entered, the next thing that happens is `return anotherValue;`
you have no code after the `else` block that needs executing, since your `else` block contains a `return` statement.
that means you do **NOT** need the `else` block, and it is just causing visual clutter.
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    
    return anotherValue;
}
```
- [ ] revise